src/Microsoft.ApiDesignGuidelines.Analyzers/Core/IdentifiersShouldNotHaveIncorrectSuffix.cs

```diff
@@ -56,13 +61,182 @@ public abstract class IdentifiersShouldNotHaveIncorrectSuffixAnalyzer : Diagnost
                                                                              DiagnosticSeverity.Warning,
                                                                              isEnabledByDefault: false,
                                                                              description: s_localizableDescription,
-                                                                             helpLinkUri: null,     // TODO: add MSDN url
+                                                                             helpLinkUri: HelpLinkUri,
                                                                              customTags: WellKnownDiagnosticTags.Telemetry);
 
         public override ImmutableArray<DiagnosticDescriptor> SupportedDiagnostics => ImmutableArray.Create(TypeNoAlternateRule, MemberNewerVersionRule, TypeNewerVersionRule, MemberWithAlternateRule);
 
+        internal const string AttributeSuffix = "Attribute";
+        internal const string CollectionSuffix = "Collection";
```

Comment:
Fixed.

---
In reply to: [57399973](https://github.com/dotnet/roslyn-analyzers/pull/885#discussion_r57399973) [](ancestors = 57399973)